spec/services/swiss_implementation_spec.rb

```diff
@@ -49,9 +49,14 @@
     end
 
     context 'with some matchups excluded' do
+      # Pull a list of just the ids and any Byes.
+      def exclude_list(players)
+        players.map { |p| p == SwissImplementation::Bye ? p : p.id }
+      end
+
       before do
-        player1.exclude = (players - [player0, player1])
-        player2.exclude = [player1]
+        player1.exclude = (exclude_list(players) - exclude_list([player0, player1]))
+        player2.exclude = exclude_list([player1])
```

Comment:
Please can we just set the data here rather than adding logic to adjust it? It's difficult to see the end result.